bin/deployment-rds.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+set -ex
+
+export AWS_ACCESS_KEY_ID=$bamboo_AWS_ACCESS_KEY_ID
+export AWS_SECRET_ACCESS_KEY=$bamboo_AWS_SECRET_ACCESS_KEY
+export AWS_DEFAULT_REGION=$bamboo_AWS_DEFAULT_REGION
+
+#clone cumulus orca template for deploying cumulus and orca
+git clone https://git.earthdata.nasa.gov/scm/orca/cumulus-orca-deploy-template.git
```

Comment:
See previous notes on checking out just the branch we want

Reply:
updated to clone only the release branch